api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -80,20 +66,4 @@ private static RetryTemplate retryTemplate() {
     tmpl.setThrowLastExceptionOnExhausted(true);
     return tmpl;
   }
-
-  private static GoogleCredential getGoogleCredential() {
-    try {
-      String saKeyPath = "src/main/webapp/WEB-INF/gsuite-admin-sa.json";
```

Comment:
Sorry, I missed your original request from me. Not sure why we were doing things this way - reading from GCS seems totally fine. Either as part of this, or as a follow-up cleanup ticket, I would also teardown the code that bundles the keyfile into the application itself: https://github.com/all-of-us/workbench/blob/dee66cf03b19209b35d087c35adeb8f4ce893ec8/api/libproject/devstart.rb#L344

Reply:
Thanks for the sanity-check. I'll flag that task as part of the overall private-key-storage cleanup (just filed as RW-4250).